README.md

```diff
@@ -1 +1,200 @@
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Set up Hello Microverse Project <a name="about-project"></a>
+
+> This is my project to practics this tools
+
+<u><b>Set up Hello Microverse Project</b></u> is a Simple excersie project to master all of our tools and best practices that we learned about in previous steps to use them in all Microverse projects.
+
+## 🛠 Built With <a name="built-with"></a>
+___
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Style</summary>
+  <ul>
+    <li><a href="">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Linters</summary>
+  <ul>
+    <li><a href="">Linters</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+____
+> Describe between 1-3 key features of the application.
+
+- **[Added index.html file]**
+- **[Added style.css file]**
+- **[Added Linter]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+___
+
+- Coming Soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+_____
+
+> To get a local copy up and running, follow these steps..
+
+
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+
+### Install
+
+ Web Browser
+
+
+
+
+### Usage
+
+To run the project, execute the following command:
+
+Code Editor
+
+### Run tests
+
+To run tests, run the following link:
+Run github link 
+
+
+### Deployment
+
+Comming Soon
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+____
+👤 **Abenezer Addisu**
+
+- GitHub: [@abenezer-addisu](https://github.com/abenezer-addisu)
+- Twitter: [@abenezer-addisu](https://twitter.com/abenezer-addisu)
+- LinkedIn: [abenezer-addisu](https://linkedin.com/in/abenezer-addisu)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+___
+
+- [ ] **[Adding Nav bar]**
+- [ ] **[Adding Footer]**
+- [ ] **[Adding more Styles]**
+
```

Comment:
[OPTIONAL] You did great 😊  adding this section. Please remove the **adding** you have before feature. You can just input only the feature. Example Navbar, Footer, UI(styles)